src/app/(protected)/notebooks/page.tsx

```diff
@@ -60,9 +97,12 @@ export default function Page() {
           <div>
             <h1 className='text-2xl pl-2 pb-8 font-bold'>My Notebooks</h1>
             <div className='flex flex-row grow'>
-            <SearchBar onChange={(e) => doFilter(e.currentTarget.value)} />              
-            <div className='flex flex-row grow justify-end'>
-                <button type="button" className="button bg-new font-medium rounded-lg text-sm px-10 py-2">New</button>
+              <SearchBar onChange={(e) => doFilter(e.currentTarget.value)} />
+              <div className='flex flex-row grow justify-end'>
+                <CreateNotebookCard
```

Comment:
Is `CreateNotebookCard` used both for the button and card? Should it be used for both like that or perhaps renamed? I don't feel like it's right. I don't think it was you who did that but I think it's worth changing this. 

Instead here we could have a component to reuse the code to open them modal but not choose between what calls it (button or card button). Hopefully you understand what I mean.